### ISSUE TITLE
Cannot find module '@babel/generator'

### ISSUE THREAD

**Issue by kevinwang5658** [logtail/logtail-js#50]
I've been exploring logtail and the functionality looks good for my use case. However when I tried to install `@logtail/node` I ran into a typescript compilation issue with babel. I can fix this issue by additionally specifying dependencies for `@babel/core`, `@babel/generator`, `@babel/template`, `@types/babel__generator`, `@types/bablel__template` but I would prefer not having to do that.

<img width="1188" alt="image" src="https://user-images.githubusercontent.com/20214115/225785876-98763501-7aca-4d81-b3a1-44a2b618eb97.png">

Typescript version: 4.6.3 (I tried on other typescript versions as well but to no avail)
Logtail version: 0.3.1

tsconfig.json:
```
{
  "compilerOptions": {
    "outDir": "build",
    "rootDir": "src",
    "esModuleInterop": true,
    "lib": ["es2020"],
    "module": "commonjs",
    "target": "es2019",
    "alwaysStrict": true,
    "noImplicitAny": true,
    "removeComments": true,
    "sourceMap": true,
  },
  "include": ["src/**/*.ts"],
  "exclude": ["build", "node_modules"],
  "watchOptions": {
    "watchFile": "useFsEvents",
    "watchDirectory": "useFsEvents",
    "fallbackPolling": "dynamicPriority",
    "synchronousWatchDirectory": true,
    "excludeDirectories": ["./node_modules", "./build"]
  },
}
```

dependencies:
```
"dependencies": {
    "express": "^4.18.1",
    "express-validator": "^6.14.0",
    "http-errors": "^2.0.0",
    "http-status-codes": "^2.2.0",
    "source-map-support": "^0.5.21",
    "uuid": "^8.3.2",
    "@logtail/node": "^0.3.1"
  },
  "devDependencies": {
    "@types/express": "^4.17.12",
    "@types/express-serve-static-core": "^4.17.21",
    "@types/http-errors": "^1.8.2",
    "@types/morgan": "^1.7.37",
    "@types/node": "^17.0.25",
    "@typescript-eslint/eslint-plugin": "^5.19.0",
    "@typescript-eslint/parser": "^5.19.0",
    "eslint": "^8.14.0",
    "eslint-plugin-import": "^2.26.0",
    "tsc-watch": "^5.0.3",
    "typescript": "^4.6.3"
  }
```

**Comment by curusarn** [comment]
Hi @kevinwang5658,

Thanks for the detailed report. I have just released a new version with fixed dependencies: `@logtail/node v0.3.3` 


Could you try it out and let me know if it solves the issue for you? 🙏
Feel free to reopen if the issue persists.

**Comment by kevinwang5658** [comment]
@curusarn Awesome works! Thank you
